scripts/cve/create_cve_trackers.py

```
@@ -0,0 +1,566 @@
+
+        if tracker_key and not args.no_link:
+            child_keys = [issue["key"] for issue in info.issues]
+            linked += link_issues(client, tracker_key, child_keys, args.dry_run)
+
+        # Update description to include the dynamic linkedIssues() link
+        if tracker_key:
+            try:
+                full_description = build_description(info, base_url=jira_url, tracker_key=tracker_key)
+                client.update_issue(tracker_key, {"description": full_description})
+                print("  Updated description with dynamic JQL link")
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)